src/metabase/driver/postgres.clj

```diff
@@ -342,14 +344,27 @@
     (pretty [_]
       (format "%s::%s" (pr-str expr) (name psql-type)))))
 
+(defn- json-query [identifier nfc-path]
+  (letfn [(handle-name [x] (if (number? x) (str x) (name x)))]
+    (apply hsql/call [:json_extract_path_text
+                      (hx/cast :json (keyword (first nfc-path)))
```

Comment:
I apologize i haven't read the product docs, but i'm assuming based on this that the first element of the path is the underlying json field? If so, why are we casting to json?

Reply:
We cast to json because it's not happy with jsonb type